addons/controllers/testdata/test-cluster-bootstrap.yaml

```diff
@@ -20,6 +20,20 @@ metadata:
   labels:
     tkg.tanzu.vmware.com/cluster-name: test-cluster-tcbt
     tanzuKubernetesRelease: v1.22.3
+spec:
+  infrastructureRef:
+    kind: VSphereCluster
+  clusterNetwork:
+    pods:
+      cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
+    services:
+      cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
+  topology:
+    class: test-clusterclass-tcbt
+    version: v1.22.3
+    variables:
+      - name: tkg.tanzu.vmware.com/cni
```

Comment:
Look like the CLI team hasn't finalized the naming convention for clusterClass vars.
Saw something in Anuj's [PR](https://github.com/vmware-tanzu/tanzu-framework/pull/1668/files#diff-64eae10d16908dfcb6b093a12243425fbb7b16e7c8c0d8573a883b9ea39a371cR41)

Reply:
Should be fine as long as the name is configurable. I think the var names will differ between TKGm and TKGS due to TKGm needing to support legacy configs.